tests/conftest.py

```diff
@@ -12,23 +24,6 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(pytest.mark.e2e)
 
 
-@pytest.fixture(params=[True, False])
-def run_with_both_engines(request, monkeypatch):
```

Comment:
this was also unused, and we now have really nice marks for v0/v1 with our tests